vision.md

```diff
@@ -0,0 +1,115 @@
+
+Implementing all those functionalities on has very high complexity.  Furthermore, more and more companies require devices to be more flexible and host additional software which might be needed to offer new digital services to customers., while taking into account non-functional requirements such as low resource footprint, security and robustness.
+
+The combination of those challenges often leads a series of custom developed embedded software that is expensive to maintain and extend. Also, most of the development used to be specific to one cloud or IoT platform. At the same time for more and more use-cases moving logic and analytics to the embedded edge device becomes a must for reasons such latency, security or cost. However, moving, and orchestrating workloads on the Edge used to be very challenging as it also requires a lot of custom logic to be developed on the device, to be able to integrate and support various device management platforms.
+
+**Example applications scenarios:** 
+Edge/embedded devices are critical compontent of any connected asset/smart equipment or operator use case. Within the different use-cases and application scenarios, edge can take over different roles to address different IoT challnges: 
+
+-   Edge devices as machine gateway:
+	- A typical problem for the target personas is the integration of  various asset specific OT interfaces to establish connection to fieldbus or industry protocols. 
+- Supporting IoT and other Northbound connectivity 
+	- Vendor agnostic connectivtiy of the data plane and control plane, analytics and service/app orchestration from various IoT platforms is required for all future IoT use-cases, as hyperscaler platforms and end-customer vendor preferences might vary. 
+-   Edge devices as deployment option for IoT services and applications 
+	- There is increasing need for device specifc applications close to the device e.g. device configuration, control logic, local monitoring, here a flexible software management framework is needed which is independent from the preferred sw artefact type due to different hardware, OS variants and package managers. 
+-   There is an emerging needs for Edge analytics such as data filtering, pre-aggregation, ML model execution. 
+-   Edge devices as configuration/management interface for asset - local/remote UI for asset configuration/management including software and firmware management of underlying systems to keep them secure and up-to date
+
+## What is it and why thin-edge.io is a game changer?
+
+To address the problems above, we created thin-edge.io as a framework for lightweight IoT gateways, PLC’s, routers and other embedded devices which require integration and interoperability with IoT platforms.
+
+The framework includes modules for cloud connectivity, data mapping, device management, intra-edge communication, and certificate management, all aspects and challenges our target personas need to address.
```

Comment:
Is 'data mapping' an actual user need, or is it more of an implementation detail?

```suggestion
The framework includes modules for cloud connectivity, data mapping, device management, intra-edge communication, and certificate management, all aspects and challenges our target personas need to address.
```